### ISSUE TITLE
[matplotcppcpp] build failure - Cannot link to openGL backend

### ISSUE THREAD

**Issue by ghost** [microsoft/vcpkg#20969]
**Host Environment**
- OS: Windows 10
- Compiler: Visual Studio 16.11.3

`./vcpkg install matplotcppcpp[opengl]`

**Failure logs** 
```
1>C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\shared\minwindef.h(130,1): warning C4005: 'APIENTRY': macro redefinition
1>C:\vcpkg\installed\x64-windows-static\include\glad\glad.h(32): message : see previous definition of 'APIENTRY'
1>ConsoleApplication5.obj : error LNK2019: unresolved external symbol "public: __cdecl matplot::backend::opengl::opengl(void)" (??0opengl@backend@matplot@@QEAA@XZ) referenced in function "void __cdecl std::_Construct_in_place<class matplot::backend::opengl>(class matplot::backend::opengl &)" (??$_Construct_in_place@Vopengl@backend@matplot@@$$V@std@@YAXAEAVopengl@backend@matplot@@@Z)
1>ConsoleApplication5.obj : error LNK2019: unresolved external symbol "public: __cdecl matplot::backend::opengl::~opengl(void)" (??1opengl@backend@matplot@@QEAA@XZ) referenced in function "public: void * __cdecl matplot::backend::opengl::`scalar deleting destructor'(unsigned int)" (??_Gopengl@backend@matplot@@QEAAPEAXI@Z)
1>ConsoleApplication5.obj : error LNK2019: unresolved external symbol "public: static double __cdecl matplot::backend::opengl::get_time(void)" (?get_time@opengl@backend@matplot@@SANXZ) referenced in function main
```

Build passed but the compiler could not find the correct symbols in the generated .lib file to link the openGL implementations.

**Comment by JonLiu1993** [comment]
@dahubley ，Does this problem still exist?

**Comment by JonLiu1993** [comment]
Closing this issue for now since there is no active thread for a long time. Please feel free to reopen this issue if it is still a problem for you.

**Comment by unageek** [comment]
I have exactly the same issue with the current master b3e0a1e625a0cc00073efb763e39088e5b75939e. Please reopen this.

**Comment by JonLiu1993** [comment]
@unageek, please provide the full error log.

**Comment by unageek** [comment]
As @dahubley said above, the problem is that `buildtrees\matplotplusplus\x64-windows-*\source\matplot\matplot_opengl.lib` is not copied to the `installed` directory and is not linked against.

**Comment by JonLiu1993** [comment]
Fixed by PR #32227
